extensions/tags/views/frontend/content/tag.blade.php

```diff
@@ -1,8 +1,8 @@
 @inject('url', 'Flarum\Http\UrlGenerator')
 
 <div class="container">
-    <h2>{{ $tag->name }}</h2>
-    <p>{{ $tag->description }}</p>
+    <h1>{{ $tag->name }}</h1>
+    <h2>{{ $tag->description }}</h2>
```

Comment:
I would revert this, description should remain a paragraph rather than a header

```suggestion
    <h1>{{ $tag->name }}</h1>
    <p>{{ $tag->description }}</p>
```

Reply:
Might be good to add `role="doc-subtitle"` to the `<p>` tag too :)

Reply:
I was wondering that for the logo element actually @davwheat and researched that (any thoughts?), thanks for providing this one, very useful!